frontend/src/components/Agreements/AgreementDetailsCards/AgreementValuesCard.jsx

```diff
@@ -0,0 +1,95 @@
+import PropTypes from "prop-types";
+import { ResponsiveBar } from "@nivo/bar";
+import CurrencySummaryCard from "../../UI/CurrencySummaryCard/CurrencySummaryCard";
+import { fiscalYearFromDate } from "../../../helpers/utils";
+
+/**
+ * A component that displays the total budget lines for an agreement.
+ *
+ * @param {Object} props - The component props.
+ * @param {Array<any>} props.budgetLineItems - The budget line items for the agreement.
+ * @returns {React.JSX.Element} - The agreement total budget lines card component JSX.
+ */
+const AgreementTotalBudgetLinesCard = ({ budgetLineItems }) => {
+    const headerText = "Total Agreement Value";
+    const valuedBlisFy = budgetLineItems.map((bli) => ({ ...bli, fiscalYear: fiscalYearFromDate(bli.date_needed) }));
+    const fyValuesMap = valuedBlisFy.reduce((acc, cur) => {
+        if (!cur.fiscalYear || cur.amount == null) return acc;
+        if (!(cur.fiscalYear in acc)) {
+            acc[cur.fiscalYear] = cur.amount;
+        } else {
+            acc[cur.fiscalYear] = acc[cur.fiscalYear] + cur.amount;
+        }
+        return acc;
+    }, {});
+    const fyValues = Object.keys(fyValuesMap).map((fy) => ({ fiscalYear: fy, amount: fyValuesMap[fy] }));
+    const totalValue = fyValues.reduce((acc, cur) => acc + cur.amount, 0);
+    const currentFiscalYear = fiscalYearFromDate(new Date());
+    const nextThreeFyValues = fyValues.filter((fyVal) => {
+        return fyVal.fiscalYear >= currentFiscalYear && fyVal.fiscalYear < currentFiscalYear + 3;
+    });
+
+    const barChartColors = [
```

Comment:
Do we want to reference global colors instead or store these somewhere where they can be referenced easier?

Reply:
great idea @tdonaworth

Reply:
done